src/sql/workbench/parts/editData/browser/editData.component.ts

```diff
@@ -409,7 +409,7 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 	 * Replace the line breaks with space.
 	 */
 	private spacefyLinebreaks(inputStr: string): string {
-		return inputStr.replace(/(\r\n|\n|\r)/g, ' ');
+		return inputStr.replace(/(\r\n|\n|\r)/g, '\u0000');
```

Comment:
What if the user has these characters in their data?

Reply:
I tested that scenario and it works as expected, it doesn't replace the user entered \u0000.

Reply:
How? If they have that exact character in their string I don't see how we would possibly be not replacing those instances.